packages/form-js-editor/test/spec/features/properties-panel/PropertiesPanel.spec.js

```diff
@@ -31,6 +31,7 @@ import {
 import schema from '../../form.json';
 import defaultValuesSchema from '../../defaultValues.json';
 import redundantValuesSchema from '../../redundantValues.json';
+import iframeSchema from '../../../../../form-js-viewer/test/spec/iframes.json';
```

Comment:
I'm wondering if generally we should start centralizing our JSON files on a single project, instead of copying across the same JSON for the editor, viewer and playground.

Reply:
Yeah I thought about a `form-js-testing` package already, could also give good guidance around testing forms in general.

Reply:
I had plans on doing that on Tasklist, I would just call it `mocks` instead of testing (unless we want to include some utility functions too)